### ISSUE TITLE
Automate Issue Management for Project Swarm Board

### ISSUE THREAD

**Issue by mariorecinos** [LittleTechBookClub/swarm-rest-api#23]
We need to automate the workflow of managing issues on our GitHub project board. Specifically, when issues are created, they should be added to the "To Do" column. When an issue is assigned, it should move to the "In Progress" column. Finally, when an issue is closed, it should move to the "Done" column.

**Acceptance Criteria:**

1. Issues are automatically added to the "To Do" column upon creation.
2. Issues move to the "In Progress" column when assigned to a user.
3. Issues move to the "Done" column when closed.
4. The workflow uses GitHub Actions to automate these transitions.
5. The setup is documented and tested to ensure it functions correctly.

**Comment by mariorojasac** [comment]
Completed in workflow project board settings.

1. Issues are automatically added to the "To Do" column upon creation.
2. Issues move to the "Done" column when closed.

**Comment by mariorecinos** [comment]
@mariorojasac When a issue is created it is in the no status column which is fine.  The second point 2. Issues move to the "In Progress" column when assigned to a user is still pending completion.  Keeping ticket open.

**Comment by fsantamaria1** [comment]
@mariorojasac, I don't think you can automate moving tickets to `In Progress` when they are assigned using the project workflows, but you can use GitHub Actions as described here:
https://docs.github.com/en/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards
